proto/README

```diff
@@ -28,7 +28,7 @@ MPS7 transaction log specification:
 
 Header:
 
-| 4 byte magic string "MPS7" | 1 byte version | 4 byte (uint32) # of records |
+| 4 byte magic string "MPS7" | 1 byte version | 4 byte (uint32) # of records minus 1 |
```

Comment:
Another way to phrase this is "the index of the last record". That might be clearer.